austin/format/speedscope.py

```diff
@@ -162,7 +162,7 @@ def asdict(self) -> SpeedscopeJson:
         }
 
     def dump(self, stream: TextIO) -> None:
-        """Dump the pprof protobuf message to the given binary stream."""
+        """Dump the JSON to a text stream."""
```

Comment:
More brainless copy-pasta from me :disappointed:

Reply:
No worries; I've definitely done that myself!